spec/factories/activity.rb

```diff
@@ -129,6 +138,14 @@
           create_list(:implementing_organisation, evaluator.implementing_organisations_count, activity: project_activity)
         end
       end
+
+      trait :newton_funded do
+        parent factory: [:fund_activity, :newton]
```

Comment:
I thought the parent of a project must be a programme. Shouldn't the parent of a `trait :newton_funded` project be a `:programme_activity, :newton_funded`?

Reply:
Ah, you're right. Put that down to ignorance on my part. I missed the distinction between project and programme early on.